src/modules/job-manager/util.c

```diff
@@ -0,0 +1,114 @@
+/************************************************************\
+ * Copyright 2018 Lawrence Livermore National Security, LLC
+ * (c.f. AUTHORS, NOTICE.LLNS, COPYING)
+ *
+ * This file is part of the Flux resource manager framework.
+ * For details, see https://github.com/flux-framework.
+ *
+ * SPDX-License-Identifier: LGPL-3.0
+\************************************************************/
+
+/* active - manipulate active jobs in the KVS
```

Comment:
minor nit: I assume this line is left over from when this file was `active.c`, should the comment read `util - manipulate active jobs in kvs` or similar?

Reply:
Oops! ! Yes sloppy error.